prog/Summarize_building_permits.sas

```diff
@@ -0,0 +1,71 @@
+/**************************************************************************
+ Program:  Summarize_building_permits.sas
+ Library:  DCRA
+ Project:  NeighborhoodInfo DC
+ Author:   Rob Pitingolo
+ Created:  03/19/19
+ Version:  SAS 9.4
+ Environment:  Windows
+ 
+ Description:  Summarize building permits by geography.
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( DCRA )
+%DCData_lib( OCTO )
+
+%let start_yr = 2009;
+%let end_yr = 2018;
+
+%let summaryvars = permit_&year. permit_construction_&year. permits_homeoccupation_&year. permits_postcard_&year. permits_shopdrawing_&year. permits_supplemental_&year.;
+
+%macro permits_by_geo (geo);
+
+proc summary data= permits_&year._geo;
+	class &geo.;
+	var &summaryvars.;
+	output out= permits_sum_&geo._&year._in sum=;
+run;
+
+data permits_sum_&geo._&year. ;
+	set permits_sum_&geo._1;
+	if _type_=1 ;
+	drop _type_ _freq_;
+
+run;
+
+%Finalize_data_set( 
+  data=permits_sum_&geo._&year.,
+  out=permits_sum_&geo._&year.,
+  outlib=DCRA,
+  label="Building permit by type statistics for DC, MD, VA and WV",
```

Comment:
DC only. Also, you should include the geography in the data set label. (See the ACS summary files as an example.) You can generate the appropriate geography label with this code, and then append onto the main label. 

```
    %let geodlbl = %sysfunc( putc( %upcase(&geo), $geodlbl. ) );
 ```